src/social-network-provider/twitter.com/index.ts

```diff
@@ -16,5 +17,15 @@ export const sharedSettings: SocialNetworkWorkerAndUI = {
     init() {},
     publicKeyEncoder: (text: string) => `🎭${text}🎭`,
     publicKeyDecoder: (text: string) => regexMatch(text, /(🎭)(.+)(🎭)/, 2),
+    payloadEncoder: (text: string) =>
+        `https://google.com/${batchReplace(text, [['🎼', '%20'], [':||', '%40'], ['+', '-'], ['=', '_'], ['|', '.']])}`,
```

Comment:
Maybe you should leave a comment about what is the transform rule

Reply:
fixed at stage 4: 851d53f2283ea4f3d05b7cdca1b5091807f9c605